packages/keyring-controller/package.json

```diff
@@ -32,6 +32,7 @@
     "@keystonehq/metamask-airgapped-keyring": "^0.6.1",
     "@metamask/base-controller": "workspace:^",
     "@metamask/controller-utils": "workspace:^",
+    "@metamask/eth-sig-util": "^5.0.2",
```

Comment:
Now that you've added this can you drop `eth-sig-util`?

Reply:
😶‍🌫️

Reply:
removed [here](https://github.com/MetaMask/controllers/pull/914/commits/e075f5bed33d0d40f63b9386a0dce773b3bc8166)